metadata-ingestion/src/datahub/ingestion/source/looker/looker_template_language.py

```diff
@@ -96,16 +105,24 @@ def resolve_liquid_variable(text: str, liquid_variable: Dict[Any, Any]) -> str:
         # Resolve liquid template
         return create_template(text).render(liquid_variable)
     except LiquidSyntaxError as e:
-        logger.warning(f"Unsupported liquid template encountered. error [{e.message}]")
+        report.info(
```

Comment:
This suffers from same shortcoming as that of looker constant errors and the same info is reported multiple times.